nebula_decoders/src/nebula_decoders_velodyne/decoders/vlp16_decoder.cpp

```diff
@@ -250,10 +255,8 @@ void Vlp16Decoder::unpack(const velodyne_msgs::msg::VelodynePacket & velodyne_pa
                 const float cos_rot_correction = corrections.cos_rot_correction;
                 const float sin_rot_correction = corrections.sin_rot_correction;
```

Comment:
`cos_rot_correction` and `sin_rot_correction` are unused now, please remove them.